src/main/java/TestFile.java

```diff
@@ -27,8 +27,8 @@ private static void anotherVariationOfIncorrectComparingToItself() {
     }
 
     private static void onMoreVariationOfIncorrectComparingToItself() {
-        int x = 23422;
-        if (x <= x) {
+        int x = 888;
```

Comment:
[Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.